styles.css

```diff
@@ -2,34 +2,48 @@
     --ie-blue: #01BCF2;
```

Comment:
There a lot of comments that could be made here, but I tried to preserve your original css as much as possible while making the site mobile friendly. If you have specific comments/suggestions, feel free to make them. I am currently not a css wizard, but I tried to make the css mobile first, so that the sirte designs would scale up nicely to larger screens.

Reply:
Thank you so much, I kinda put the design/css of the site on the side, this looks great!